sass/overrides/_skip-to-content.scss

```diff
@@ -0,0 +1,18 @@
+.skip-to-content {
```

Comment:
Css should live in the pattern-library. The overrides here is only for adjusting css connected to different libraries we are using. The pattern-library lives in this repository: https://github.com/ushahidi/platform-pattern-library/, you can find guidelines on how to set it up here: https://docs.ushahidi.com/platform-developer-documentation/development-and-code/setup_alternatives/setting-up-the-pattern-library-for-development

Reply:
@Angamanga, I will move the styles to the pattern library ASAP
thanks!